templater/jobs/postsubmits.go

```diff
@@ -0,0 +1,483 @@
+			RunIfChanged: "GIT_TAG",
+			Timeout:      "6h",
+			Commands:     []string{"make update-attribution-files"},
+			Resources: &Resources{
+				Requests: &ResourceConfig{
+					CPU:    "4",
+					Memory: "16Gi",
+				},
+			},
+			Volumes: []*Volume{
+				{
+					Name:       "github-auth",
+					VolumeType: "secret",
+					Secret: &Secret{
+						Name:        "pr-bot-github-token",
+						DefaultMode: 256,
+					},
+				},
+				{
+					Name:       "ssh-auth",
+					VolumeType: "secret",
+					Secret: &Secret{
+						Name:        "pr-bot-ssh-secret",
+						DefaultMode: 256,
+					},
+				},
+			},
+			VolumeMounts: []*VolumeMount{
+				{
+					Name:      "github-auth",
+					MountPath: "/secrets/github-secrets",
+					ReadOnly:  true,
+				},
+				{
+					Name:      "ssh-auth",
+					MountPath: "/secrets/ssh-secrets",
+					ReadOnly:  true,
+				},
+			},
+		},
+		"build-1-X-postsubmits.yaml": JobConfig{
+			HasReleaseBranches: true,
+			JobName:            "build-1-X-postsubmit",
+			RunIfChanged:       ".*",
+			Timeout:            "6h",
+			Commands:           []string{"build/lib/buildkit_check.sh", "cp -r \"${HOME}/.docker\" /home/prow/go/src/github.com/aws/eks-distro", "make postsubmit-conformance", "touch /status/done"},
+			ImageBuild:         true,
+			Resources: &Resources{
+				Requests: &ResourceConfig{
+					CPU:    "4",
+					Memory: "16Gi",
+				},
+			},
+			BuildkitResources: &Resources{
+				Requests: &ResourceConfig{
+					CPU:    "1",
+					Memory: "4Gi",
+				},
+			},
+			EnvVars: []*EnvVar{
+				{
+					Name:  "TEST_ROLE_ARN",
+					Value: "arn:aws:iam::125833916567:role/TestBuildRole",
+				},
+				{
+					Name:  "ARTIFACT_BUCKET",
+					Value: "eks-d-postsubmit-artifacts",
+				},
+				{
+					Name:  "RELEASE_BRANCH",
+					Value: "1-X",
+				},
+				{
+					Name:  "CONTROL_PLANE_INSTANCE_PROFILE",
+					Value: "arn:aws:iam::125833916567:instance-profile/KopsControlPlaneBuildRole",
+				},
+				{
+					Name:  "NODE_INSTANCE_PROFILE",
+					Value: "arn:aws:iam::125833916567:instance-profile/KopsNodesBuildRole",
+				},
+				{
+					Name:  "KOPS_STATE_STORE",
+					Value: "s3://testbuildstack-125833916-kopsbuildstatestorebucke-d4esen60nfrk",
+				},
+				{
+					Name:  "IMAGE_REPO",
+					Value: "public.ecr.aws/h1r8a7l5",
+				},
+				{
+					Name:  "DOCKER_CONFIG",
+					Value: "/home/prow/go/src/github.com/aws/eks-distro/.docker",
+				},
+			},
+		},
+		"build-1-21-minimal-postsubmits.yaml": JobConfig{
+			HasReleaseBranches: true,
+			JobName:            "build-1-21-minimal-postsubmit",
+			RunIfChanged:       ".*",
+			Timeout:            "6h",
+			Commands:           []string{"build/lib/buildkit_check.sh", "cp -r \"${HOME}/.docker\" /home/prow/go/src/github.com/aws/eks-distro", "make -j2 postsubmit-conformance", "touch /status/done"},
+			ImageBuild:         true,
+			Resources: &Resources{
+				Requests: &ResourceConfig{
+					CPU:    "4",
+					Memory: "16Gi",
+				},
+			},
+			BuildkitResources: &Resources{
```

Comment:
Lets standardize this based on ImageBuild:true

Reply:
Done